modules/stochastic_tools/test/tests/samplers/directperturbation/tests

```diff
@@ -0,0 +1,44 @@
+[Tests]
+  design = 'DirectPerturbationSampler.md'
+  issues = '#28671'
+
+  [central_difference]
+    requirement = 'The system shall be able to generate samples for a direct perturbation method with a central difference scheme:'
+
+    [serial]
+      type = CSVDiff
+      input = directperturbation.i
+      csvdiff = 'directperturbation_out_data_0001.csv'
+      detail = "with replicated output;"
+    []
+    [parallel]
+      type = CSVDiff
+      input = directperturbation.i
+      cli_args = 'VectorPostprocessors/data/parallel_type=DISTRIBUTED Outputs/file_base=dp_cd_3'
+      min_parallel = 3
+      max_parallel = 3
+      csvdiff = 'dp_cd_3_data_0001.csv.0 dp_cd_3_data_0001.csv.1 dp_cd_3_data_0001.csv.2'
```

Comment:
Like here I understand you are stuck with 3 processes. But for reporters?

Reply:
Yeah removed that. That reporter will always output on root, and only the data used is distributed/root. Great catch!